bitnami/redis/values.yaml

```diff
@@ -488,6 +488,15 @@ master:
     ## NOTE: requires master.persistence.enabled: true
     ##
     existingClaim: ""
+  ## persistentVolumeClaimRetentionPolicy
```

Comment:
Please ensure that the formatting is maintained. Each parameter should include a comment with "##" preceding it. You can refer to an example in these lines: [link](https://github.com/bitnami/charts/pull/19689/files#diff-5063c6dcbc478573073c2f1b280610c29b86379ce92e6dc3ce04b51eeb0c8fd6R516-R518).

Reply:
Sure, done in [this commit](https://github.com/bitnami/charts/pull/19689/commits/ed5f8c72306b878bdfa0cb57251fbbd79ebdff61).